xtask/src/cli.rs

```diff
@@ -35,9 +35,9 @@ impl Default for Subcommand {
 
 #[derive(StructOpt, Default)]
 pub struct CoverageOpts {
-    /// Creates an lcov output instead of printing to the terminal.
```

Comment:
Was anyone actually printing it to terminal? It didn't seem that useful for me, but I'm happy to put it back if necessary.

Reply:
I hadn't used in practice, so fine to remove imo!